src/Cleaners.py

```diff
@@ -0,0 +1,80 @@
+import re, pickle, os
+
+import pandas as pd
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.tokenize import word_tokenize
+from nltk.tag import pos_tag
+from nltk.corpus import stopwords
+
+class Classifier:
+    this_dir, this_filename = os.path.split(__file__) 
+    data_path = os.path.join(this_dir, 'model/pol_classifier.pickle') 
+
+class Cleaners(Classifier):
+    
+    def __init__(self):
+        model_hold = open(Cleaners.data_path, 'rb')
+        self.classifier = pickle.load(model_hold)
+        model_hold.close()
+
+    def clean_tweets(self, tweet_text, tweet_date):
```

Comment:
Don't forget to add docstrings to your functions to explain what the function does, what the input parameters are and the output of the function